### ISSUE TITLE
[BUG] uploading a filename with a comma crashes flowise

### ISSUE THREAD

**Issue by jaredmcqueen** [FlowiseAI/Flowise#2541]
**Describe the bug**

Flowise crashes when a file with a comma is uploaded.I found this using unstructured, but they apply to all the file upload types:

- Csv
- Docx
- Json
- Jsonlines
- Pdf
- Text
- UnstructuredFile

flowise expects the data to be in a `"<base64>,<filename>"` format
```ts
const value = result + `,filename:${name}`
```

On the server-side, there is splitting based on this comma, which throws an error if a filename exists.
```ts
const splitDataURI = file.split(',')
const filename = splitDataURI.pop()?.split(':')[1] ?? ''
const bf = Buffer.from(splitDataURI.pop() || '', 'base64')
const loaderDocs = await loader.loadAndSplitBuffer(bf, filename)
docs.push(...loaderDocs)

```

**To Reproduce**

upload any file with a comma.  Flowise CRASHES

**Expected behavior**
upload a file with a comma.  Have flowise not crash.

**Screenshots**
an example of a filename with a comma.  There ends up being 4 elements in an array which will cause a crash.
<img width="751" alt="Screenshot 2024-05-31 at 10 24 18 AM" src="https://github.com/FlowiseAI/Flowise/assets/3491953/9493d87e-874d-453d-808f-7ec4fc0f43a0">

Data should really be passed around using JSON.
```ts
export interface IFileObject {
    data?: string
    filename: string
}
```

I got this working on a local branch, but would have to update the UI and 7 file handlers to fix for all of flowise.  I could do, but it'd take time.  All I use right now is `unstructuredFile`, and never through the UI.

**Comment by HenryHengZJ** [comment]
@jaredmcqueen good catch! can you open a PR just for the UnstructuredFile, we can then take the fix and implement on other doc loaders
